internal/declcfg/load.go

```diff
@@ -17,28 +17,26 @@ import (
 	"github.com/operator-framework/operator-registry/internal/property"
 )
 
-func LoadDir(configDir string) (*DeclarativeConfig, error) {
-	w := &dirWalker{}
-	return loadFS(configDir, w)
-}
-
-func loadFS(root string, w fsWalker) (*DeclarativeConfig, error) {
+func LoadFS(configFS fs.FS) (*DeclarativeConfig, error) {
```

Comment:
nit: we can still call the fs `root`?

Reply:
I changed the name because I thought `configFS` was clearer than `root`, but thinking about this a second time, I agree `root` (+ some GoDoc comment text) is better.